README Fix

yuheswari opened the pull request:
Added $ prefix to shell commands in README for clarity. 
Changed 99-zealphp-swoole.ini to 99-zealphp-openswoole.ini as in setup.sh.

yuheswari committed:
added $ prefix to shell commands in README for clarity and changed 99-zealphp-swoole.ini to 99-zealphp-openswoole.ini

sibidharan commented on README.md:
```diff
@@ -24,21 +24,21 @@ We will install some dependencies before configuring OpenSwoole.
 #!/bin/bash
```
The block was marked with a shebang so we didnt include $. If we are adding $ to it, its better we remove the shebang and merge all the `apt install` into single command and adding $ in front will make much more sense.

yuheswari replied:
Thank you for the notification! I have committed the changes, kindly check it!

yuheswari committed:
Update README.md

removed shebang and  modified accordingly. Also added sudo in creating .ini file for zealphp. 

Added  instructions to ensure error free setup.
